CyborgianStates.Tests/CommandTests/CouldEndorseCommand.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace CyborgianStates.Tests.CommandTests;
+
+public class CouldEndorseCommand : BaseCommandTests
```

Comment:
What is the purpose of this empty test class ? Either remove it or implement the related unit tests for this command.

Reply:
I will remove it as comprehending the structure of existing unit tests is beyond my ability (w/out further guidance)